package.json

```diff
@@ -63,18 +70,14 @@
     "babel-runtime": "^6.3.19",
     "bootstrap": "^3.3.6",
     "classnames": "^2.1.5",
+    "cloudgov-style": "git+https://git@github.com/18F/cg-style.git#master",
```

Comment:
I think we could. I've been pretty good about using semantic versioning on `cg-style`, and we'll end up using it to the extent where we'll need a good process to do things like this.

This webpack is a big change, which changes a lot of cg-style, so I'm doing things in a slower manner. My goal with cg-style is: you see a need for a change in cg-deck, you make the change in cg-style, you npm link to ensure the change worked, you test cg-style to make sure nothing broke and then you push up a new version. So it'll be quick and cg-deck will mostly use the new versions of cg-style.

That's my ideal, and we can see how well that pans out.

But for this PR, which I plan in updating a minor, cg-style to 0.5, it's a little slower.